README.md

```diff
@@ -35,7 +35,7 @@ Once you have cloned the repository, you can open the project in Unity. You can
 
 ### Contributing
 
-To start, create a branch in GitKraken by clicking the "Branch" button. Then, enter a name for your branch and click "Create Branch". You can now make changes to the project. Once you are done, you can commit your changes by clicking the "Commit" button. Then, enter a message for your commit and click "Commit to Branch". You can now push your changes to the repository by clicking the "Push" button.
+To start, create a branch in GitKraken or Github Desktop by clicking the "Branch" button. Then, enter a name for your branch and click "Create Branch". You can now make changes to the project. Once you are done, you can commit your changes by clicking the "Commit" button. Then, enter a message for your commit and click "Commit to Branch". You can now push your changes to the repository by clicking the "Push" button.
```

Comment:
```suggestion
To start, create a branch in GitKraken by clicking the "Branch" button. Then, enter a name for your branch and click "Create Branch". You can now make changes to the project. Once you are done, you can commit your changes by clicking the "Commit" button. Then, enter a message for your commit and click "Commit to Branch". You can now push your changes to the repository by clicking the "Push" button.
```
GitHub Desktop uses a different setup for creating and managing branches, so these steps are invalid.

Reply:
```suggestion
To start, create a branch in GitKraken by clicking the "Branch" button. Then, enter a name for your branch and click "Create Branch". You can now make changes to the project. Once you are done, you can commit your changes by clicking the "Commit" button. Then, enter a message for your commit and click "Commit to Branch". You can now push your changes to the repository by clicking the "Push" button. Similar steps can be followed using GitHub Desktop.
```
I'd rather make sure our documentation is as specific as possible. Freshmen and people unfamiliar with version control may be working on this project. I'm okay with just saying that similar steps exist for GitHub Desktop if that works with you.